ios/FluentUI/Tooltip/TooltipPositionController.swift

```diff
@@ -33,14 +33,17 @@ class TooltipPositionController {
     func updateArrowDirectionAndTooltipSize(for message: String, title: String? = nil, tokenSet: TooltipTokenSet) {
         let preferredBoundingRect = boundingRect.inset(by: anchorViewInset(for: preferredArrowDirection))
         let backupBoundingRect = boundingRect.inset(by: anchorViewInset(for: preferredArrowDirection.opposite))
-        let preferredSize = TooltipView.sizeThatFits(preferredBoundingRect.size,
+        let isAccessibilityContentSize = window.traitCollection.preferredContentSizeCategory.isAccessibilityCategory
+        let preferredSize = TooltipViewController.sizeThatFits(preferredBoundingRect.size,
```

Comment:
fundamentally sizeThatFits on viewcontroller doesn't make sense. why do we still need toptippostioncontroller separate from your newly tooltipviewcontroller.swift?

Reply:
discussed offline, this will be handled in a separate PR!